upgrade/1.2/plan_and_coordinate_network_upgrade.md

```diff
@@ -97,18 +96,19 @@ referring to user activity.
    ```
 
 1. Once UAN has been upgraded to 2.4, the UANs may be rebooted for the new network configuration changes to take effect.
-   UANs will not receive an IP address on the CMN network and instead will default their traffic through the new CAN/CHN. For concrete
-   details on UAN transition plan for users, see
+   UANs will not receive an IP address on the CMN network and instead will default their traffic through the new CAN/CHN.
+   For concrete details on UAN transition plan for users, see
    [Minimize UAN Downtime](../../operations/network/management_network/bican_enable.md#minimize-uan-downtime).
 
 1. Note that in CSM 1.2, UAN ports will not be removed from the CMN VLAN7 in switches. In the next CSM release, switch
    configuration will be updated to remove UAN ports from the CMN VLAN7. This enables non-rebooted UANs to continue to work
    and allows for better easing into BICAN in CSM 1.2. For more details about this transition plan, see
    [Minimize UAN Downtime](../../operations/network/management_network/bican_enable.md#minimize-uan-downtime).
 
-## Manually removing UAN switch ports from the CMN VLAN7 (Placeholder)
+## Manually removing UAN switch ports from the CMN VLAN 7
 
-After the upgrade to UAN 2.4, the UAN switch ports should be removed from CMN VLAN7 to prevent user traffic from being able to reach endpoints on the CMN.  In the CSM 1.2 and UAN 2.4 upgrade, this removal is not done automatically. A future release or hotfix to CSM will introduce this automation.
+After the upgrade to UAN 2.4, the UAN switch ports should be removed from CMN VLAN 7 to prevent user traffic from being able to reach endpoints on the CMN.
```

Comment:
```suggestion
After the upgrade to UAN 2.4, the UAN switch ports should be removed from CMN VLAN 7, to prevent user traffic from being able to reach endpoints on the CMN.
```